Makefile

```diff
@@ -22,23 +32,17 @@ export XDG_CACHE_HOME ?= /tmp/.local/cache
 # More info on the awk command:
 # http://linuxcommand.org/lc3_adv_awk.php
 
-.PHONY: help
-help: ## Display this help.
-	@awk 'BEGIN {FS = ":.*##"; printf "Usage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+#.PHONY: help
```

Comment:
build-machinery-go has its own `help` target. Assuming we want to proceed with this, I'll push an update that removes this.